linux-tkg-config/prepare

```diff
@@ -981,7 +981,11 @@ _tkg_srcprep() {
   fi
 
   if [ "$_compiler_name" = "-llvm" ]; then
+    _enable "CFI_CLANG"
```

Comment:
I'd put that one to its default value, disabled. People can then enable it with frags if needed

Reply:
ok, thought enabled was default, my bad